x/upgrade/module.go

```diff
@@ -155,11 +156,18 @@ func (am AppModule) ExportGenesis(_ sdk.Context, cdc codec.JSONCodec) json.RawMe
 // ConsensusVersion implements AppModule/ConsensusVersion.
 func (AppModule) ConsensusVersion() uint64 { return ConsensusVersion }
 
+// PreBlock calls the upgrade module hooks
+//
+// CONTRACT: this is called *before* all other modules' BeginBlock functions
+func (am AppModule) PreBlock(ctx context.Context) (appmodule.ResponsePreBlock, error) {
+	return PreBlocker(ctx, am.keeper)
+}
+
 // BeginBlock calls the upgrade module hooks
 //
 // CONTRACT: this is registered in BeginBlocker *before* all other modules' BeginBlock functions
 func (am AppModule) BeginBlock(ctx context.Context) error {
```

Comment:
I think for consistency we should delete this one and possibly add OrderPreBlocker to the app config and module manager.

Reply:
you mean add SetOrderPreBlockers so we could prioritize upgrade pre blocker right?

Reply:
Yeah

Reply:
Roger that, will add later